### PR TITLE
AutoCorrect files/zh-cn/web/api/n*/

### PR DESCRIPTION
**Pull request by huacnlee**

### CODE REVIEW

**Comment by yin1999** [comment]
could we continue this job until all those existing jobs done.

**Review by huacnlee — COMMENTED** [review]
已检查，修正

File: files/zh-cn/web/api/namednodemap/index.html [hunk 1]
```diff
@@ -36,11 +36,11 @@ <h2 id="方法">方法</h2>
  <dt>{{domxref("NamedNodeMap.item()")}}</dt>
  <dd>返回指定索引处的属性节点（{{ domxref("Attr") }}），或者，当索引超出或等于属性节点的数量时，返回 <code>null</code>。</dd>
  <dt>{{domxref("NamedNodeMap.getNamedItemNS()")}}</dt>
- <dd>根据给定的命名空间参数和name参数返回一个{{ domxref("Attr") }}对象。</dd>
+ <dd>根据给定的命名空间参数和 name 参数返回一个{{ domxref("Attr") }}对象。</dd>
```
Comment (yin1999) [inline files/zh-cn/web/api/namednodemap/index.html:39]:
```suggestion
 <dd>根据给定的命名空间参数和 name 参数返回一个 {{domxref("Attr")}} 对象。</dd>
```

File: files/zh-cn/web/api/namednodemap/index.html [hunk 2]
```diff
@@ -36,11 +36,11 @@ <h2 id="方法">方法</h2>
...
  <dt>{{domxref("NamedNodeMap.setNamedItemNS()")}}</dt>
- <dd>替换、添加给定命名空间参数和name参数的{{ domxref("Attr") }} 对象 。</dd>
+ <dd>替换、添加给定命名空间参数和 name 参数的{{ domxref("Attr") }} 对象 。</dd>
```
Comment (yin1999) [inline files/zh-cn/web/api/namednodemap/index.html:41]:
```suggestion
 <dd>替换、添加给定命名空间参数和 name 参数的 {{domxref("Attr")}} 对象 。</dd>
```

File: files/zh-cn/web/api/namednodemap/index.html [hunk 3]
```diff
@@ -36,11 +36,11 @@ <h2 id="方法">方法</h2>
...
  <dt>{{domxref("NamedNodeMap.removeNamedItemNS()")}}</dt>
- <dd>删除给定命名空间参数和name参数的{{ domxref("Attr") }} 对象 。</dd>
+ <dd>删除给定命名空间参数和 name 参数的{{ domxref("Attr") }} 对象 。</dd>
```
Comment (yin1999) [inline files/zh-cn/web/api/namednodemap/index.html:43]:
```suggestion
 <dd>删除给定命名空间参数和 name 参数的 {{domxref("Attr")}} 对象 。</dd>
```

File: files/zh-cn/web/api/navigation_timing_api/index.html [hunk 4]
```diff
@@ -7,33 +7,33 @@
...
+<p><strong>Navigation Timing API</strong> 提供了可用于衡量一个网站性能的数据。与用于相同目的的其他基于 JavaScript 的机制不同，该 API 可以提供可以更有用和更准确的端到端延迟数据。</p>
 
 <p> </p>
 
 <h2 id="Concepts_and_usage">Concepts and usage</h2>
 
-<p>你可以使用Navigation Timing API在客户端收集性能数据，并用{{domxref("XMLHttpRequest")}} 或其它技术传送到服务端。同时，该API使你可以衡量之前难以获取的数据，如卸载前一个页面的时间，在域名解析上的时间，在执行{{event("load")}}事件处理器上花费的总时间等。</p>
+<p>你可以使用 Navigation Timing API 在客户端收集性能数据，并用 {{domxref("XMLHttpRequest")}} 或其它技术传送到服务端。同时，该 API 使你可以衡量之前难以获取的数据，如卸载前一个页面的时间，在域名解析上的时间，在执行 {{event("load")}} 事件处理器上花费的总时间等。</p>
 
 <p> </p>
```
Comment (yin1999) [inline files/zh-cn/web/api/navigation_timing_api/index.html:19]:
```suggestion
```

File: files/zh-cn/web/api/nodelist/foreach/index.html [hunk 5]
```diff
@@ -85,7 +85,7 @@ <h2 id="Polyfill">Polyfill</h2>
    NodeList.prototype.forEach = Array.prototype.forEach;
 }</pre>
 
-<p>上面的代码是大部分浏览器实现的 <code>NodeList.prototype.forEach()</code> (例如Chrome).</p>
+<p>上面的代码是大部分浏览器实现的 <code>NodeList.prototype.forEach()</code> (例如 Chrome).</p>
```
Comment (yin1999) [inline files/zh-cn/web/api/nodelist/foreach/index.html:88]:
```suggestion
<p>上面的代码是大部分浏览器实现的 <code>NodeList.prototype.forEach()</code>（例如 Chrome）。</p>
```

File: files/zh-cn/web/api/nodelist/keys/index.html [hunk 6]
```diff
@@ -41,7 +41,7 @@ <h2 id="例子">例子</h2>
 }
 </pre>
 
-<p>结果是:</p>
+<p>结果是：</p>
 
 <pre class="notranslate">0
```
Comment (yin1999) [inline files/zh-cn/web/api/nodelist/keys/index.html:46]:
```suggestion
<pre>0
```

File: files/zh-cn/web/api/navigation_timing_api/index.html [hunk 7]
```diff
@@ -7,33 +7,33 @@
 ---
 <p>{{DefaultAPISidebar("Navigation Timing")}}</p>
 
-<p><strong>Navigation Timing API</strong> 提供了可用于衡量一个网站性能的数据。与用于相同目的的其他基于JavaScript的机制不同，该API可以提供可以更有用和更准确的端到端延迟数据。</p>
+<p><strong>Navigation Timing API</strong> 提供了可用于衡量一个网站性能的数据。与用于相同目的的其他基于 JavaScript 的机制不同，该 API 可以提供可以更有用和更准确的端到端延迟数据。</p>
 
 <p> </p>
```
Comment (yin1999) [inline files/zh-cn/web/api/navigation_timing_api/index.html:13]:
```suggestion
```

**Pull request merged**